# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iptables/complier.h.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- iptables-1.4.0/include/linux/netfilter_ipv4/ip_tables.h.vanilla	2008-03-17 11:30:07.000000000 +0100
+++ iptables-1.4.0/include/linux/netfilter_ipv4/ip_tables.h	2008-03-17 11:31:55.000000000 +0100
@@ -15,7 +15,7 @@
 #ifndef _IPTABLES_H
 #define _IPTABLES_H
 
-#include <linux/compiler.h>
+#define __user
 #include <linux/netfilter_ipv4.h>
 
 #include <linux/netfilter/x_tables.h>
--- iptables-1.4.0/include/linux/netfilter_ipv6/ip6_tables.h.vanilla	2008-03-17 11:30:25.000000000 +0100
+++ iptables-1.4.0/include/linux/netfilter_ipv6/ip6_tables.h	2008-03-17 11:31:47.000000000 +0100
@@ -15,7 +15,7 @@
 #ifndef _IP6_TABLES_H
 #define _IP6_TABLES_H
 
-#include <linux/compiler.h>
+#define __user
 #include <linux/netfilter_ipv6.h>
 
 #include <linux/netfilter/x_tables.h>
